src/Provider/ImageProvider.php

```diff
@@ -127,17 +127,6 @@ public function getHelperProperties(MediaInterface $media, $format, $options = [
         return array_merge($params, $options);
     }
 
-    /**
-     * {@inheritdoc}
-     */
-    public function getReferenceImage(MediaInterface $media)
```

Comment:
Method is public and removing it would be a BC Break, you can deprecate it if you want.

Reply:
Why would it be a BC break if the parent method exists?

Reply:
My bad, didn’t understand